components/pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/Pagination.scss';
+
+export type PaginationProps = StandardProps & {
+  results: number
+  resultsPerPage: number
+  page: number
+  previous?: string
+  pages?: number[]
+  next?: string
```

Comment:
What are these? - They don't seem to be used.

Reply:
They aren't. I included them as I had expected to use them later but turns out they aren't needed. I have now removed these.